modules/apps/object/object-api/src/main/java/com/liferay/object/constants/ObjectPortletKeys.java

```diff
@@ -22,4 +22,8 @@ public class ObjectPortletKeys {
 		"com_liferay_object_web_internal_object_entries_portlet_" +
```

Comment:
Hey @marcelabc, as far as I know, [this](https://github.com/liferay-objects/liferay-portal/blob/66aa670df749419197a5e745724fff521c225834/modules/apps/object/object-api/src/main/java/com/liferay/object/constants/ObjectPortletKeys.java#L13-L15) objectPortletKey is pointing to a wrong path. Does the `ListTypeDefinitionsPortlet` really needs two `"portlet_"` in its path? Should we fix this?

Reply:
Touching this would probably have ramifications and we would have to do upgrades for Resources and ResourcePermissions on the database. I don't think we should touch this for now